superheroes/superhero-06-01/api/utils/gemini_api.py

```diff
@@ -0,0 +1,16 @@
+import google.generativeai as genai
+from environs import Env
+
+env = Env()
+env.read_env()
+genai.configure(api_key = env.str("API_KEY"))
```

Comment:
This will break. As we don't have yet an alternative for secrets, we should put the API_KEY hardcoded for now 🙃.

Reply:
Ok